Makefile

```diff
@@ -66,10 +66,71 @@ build-all:
 aro: check-release generate
 	go build -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro
 
-.PHONY: runlocal-rp
-runlocal-rp:
-	go run -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro rp
+# Target to create Podman secrets
+.PHONY: podman-secrets
+podman-secrets: aks.kubeconfig
+	podman secret rm --ignore aks.kubeconfig
+	podman secret create aks.kubeconfig ./aks.kubeconfig
+
+	podman secret rm --ignore proxy-client.key
+	podman secret create proxy-client.key ./secrets/proxy-client.key
+
+	podman secret rm --ignore proxy-client.crt
+	podman secret create proxy-client.crt ./secrets/proxy-client.crt
 
+	podman secret rm --ignore proxy.crt
+	podman secret create proxy.crt ./secrets/proxy.crt
+
+# Target to run the local RP
+.PHONY: runlocal-rp
+runlocal-rp: ci-rp podman-secrets
+	@set -a; source secrets/env; set +a; \
+	podman run --rm -p 127.0.0.1:8443:8443 \
+		--name aro-rp \
+		-w /app \
+		-e RP_MODE="development" \
+		-e PROXY_HOSTNAME="$${PROXY_HOSTNAME}" \
```

Comment:
for linux, we'll need the `$${FOO}` syntax replaced with `$(FOO)` syntax on all envvars.